protocol/synthetix/contracts/storage/RewardDistribution.sol

```diff
@@ -109,7 +124,18 @@ library RewardDistribution {
             self.lastUpdate = 0;
 
             diffD18 += updateEntry(self, totalSharesD18);
-        }
+				} else {
```

Comment:
With adding another kind of condition about scheduling, would be good to be more explicit about all the conditions, rather than just `else`, with default case throwing error of sorts.

smth like
```
if (true1) {
  do_stuff1
  return
}

if (true2) {
  do_stuff2
  return
}

revert SomethingOff
```